apps/public-reference/package.json

```diff
@@ -28,6 +28,7 @@
     "next-plugin-custom-babel-config": "^1.0.2",
     "node-polyglot": "^2.4.0",
     "node-sass": "^4.12.0",
+    "oidc-client": "^1.10.1",
```

Comment:
This is the client portion of the app. `oidc-provider` is for the auth service (since it is an OIDC server). The `oidc-client` is the client side code that implements the login flow within the SPA.

Since OIDC is a standard, any client that implements the standard correctly would work. I actually initially started writing a simple custom implementation on this, but then decided to go with a standard certified library (mostly for the "background signin" functionality).

[This page](https://openid.net/developers/certified/) lists OIDC-certified libraries. `oidc-client` is the certified JS library for browser-based apps (e.g. SPAs). `openid-client` is a library for node.js apps that need to act as open-id clients of some auth server (e.g. for making federated data requests).